### PR TITLE
Release 2.17.0

### PR DESCRIPTION
**Pull request by borisprimer**
Changelogs are in [Notion](https://www.notion.so/primerio/iOS-Changelog-2-17-0-TEMP-b6451f0fbaa443f9888838cc3b2fbe6c?showMoveTo=true&saveParent=true).

### CODE REVIEW

File: Internal/Debug App/Project.swift [hunk 1]
```diff
@@ -14,7 +14,7 @@ enum AppSettings {
 
     static let settingsDictionary = SettingsDictionary()
         .merging(BaseSettings.settingsDictionary)
-        .merging(["CODE_SIGN_IDENTITY": .string("Apple Development: DX Primer (8B5K7AGMS8)")])
+        .merging(["CODE_SIGN_IDENTITY": .string("Apple Development: MOHAMMED TARIK ALADIN TALEB (P3K69N82DT)")])
```
Comment (xevious78) [inline Internal/Debug App/Project.swift:17]: Is that required to have my name here @semirp ? :p

File: Sources/PrimerSDK/Classes/PCI/Services/Network/URLSessionStack.swift [hunk 2]
```diff
@@ -325,7 +325,11 @@ internal extension URLSessionStack {
     private func url(for endpoint: Endpoint) -> URL? {
         guard let urlStr = endpoint.baseURL else { return nil }
         guard let baseUrl = URL(string: urlStr) else { return nil }
-        let url = baseUrl.appendingPathComponent(endpoint.path)
+        var url = baseUrl
+        
+        if endpoint.path != "" {
+            url = baseUrl.appendingPathComponent(endpoint.path)
```
Comment (xevious78) [inline Sources/PrimerSDK/Classes/PCI/Services/Network/URLSessionStack.swift:331]: I'd expect some unit test for something as simple as this change @BorisNikolic

File: Internal/Debug App/Project.swift (hunk 1 above)
Comment (semirp) [reply Internal/Debug App/Project.swift:17]: yes 🗡️

File: Sources/PrimerSDK/Classes/PCI/Services/Network/URLSessionStack.swift (hunk 2 above)
Comment (borisprimer) [reply Sources/PrimerSDK/Classes/PCI/Services/Network/URLSessionStack.swift:331]: Tests are merged.

**Review by semirp — APPROVED** [review]

**Review by xevious78 — APPROVED** [review]

**Pull request merged**